core/src/main/scala/pureconfig/BasicConverters.scala

```diff
@@ -0,0 +1,106 @@
+package pureconfig
+
+import java.net.{ URI, URL }
+import java.nio.file.{ Path, Paths }
+import java.time._
+import java.util.UUID
+
+import scala.concurrent.duration.{ Duration, FiniteDuration }
+
+import com.typesafe.config._
+import pureconfig.error._
+
+/**
+ * Trait containing `ConfigConvert` instances for primitive types and simple classes in Java and Scala standard
+ * libraries.
+ */
+trait BasicConverters extends ConvertHelpers {
+
+  // primitive type converters
```

Comment:
I don't mind changing it, I'm ok either way :) @melrief and @jcazevedo, do you have any preference about this?